docker/openproblems/Dockerfile

```diff
@@ -1,6 +1,19 @@
-FROM rackspacedot/python38:latest
+FROM python:3.8
 
-ENV DEBIAN_FRONTEND=noninteractive
+# Setting up Sagemaker Studio Image from example
+# https://github.com/aws-samples/sagemaker-studio-custom-image-samples/blob/main/examples/echo-kernel-image/Dockerfile
+
+ARG NB_USER="sagemaker-user"
+ARG NB_UID="1000"
+ARG NB_GID="100"
+
+# Setup the "sagemaker-user" user with root privileges.
+RUN \
+   apt-get update && \
+   apt-get install -qy sudo && \
+   useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
+   chmod g+w /etc/passwd && \
+   echo "${NB_USER}    ALL=(ALL)    NOPASSWD:    ALL" >> /etc/sudoers
```

Comment:
if you `apt-get update` here, I would remove the apt-get folder after installation of what you need to make the image slimmer.